docs-kits/kits/Connector Kit/Adoption View/policies-in-catena.md

```diff
@@ -0,0 +1,230 @@
+Framework Agreements are roughly structured along the lines of business scenarios under which a set of business partners
+want to exchange data.
+
+Each Participant commits to a set of Framework Agreements during Onboarding. They are
+granted a set of VCs as proof of that commitment. Consequently, Framework Agreement Constraints belong to the kind
+of `odrl:Constraint`s that have to
+be [checked against a VP](working-with-policies.md#provider-side-checking-a-consumers-verifiable-presentation).
+The complete set is listed in the most current version of
+standard [CX-0050 Framework Credential](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_September23/CX-0050-FrameworkAgreementCredential-v.1.0.0.pdf) (
+link opens pdf).
+Framework Agreements are referred to in a machine-readable way in a Provider's Offers. When a Consumer starts the
+negotiation for said offer, not only will the Policy in the `ContractRequestMessage` be checked but also his
+Credentials. Here's an example of an `odrl:Constraint` referencing a Framework Agreement and invoking the VC-check:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "odrl:leftOperand": "https://w3id.org/catenax/policy/FrameworkAgreement",
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "traceability:1.0"
+}
+```
+
+### Usage Purposes
+
+Purposes are usually part of a Framework Agreement and restrict the purpose the Consumer is privileged to use the
+obtained data for. Unlike a Framework Agreement Constraint, the purposes are NOT checked against VCs, thus necessary
+for a successful negotiation mechanism is
+only [the Consumer's consent to the Offer](working-with-policies.md#consumer-side-odrloffer-in-a-contractrequestmessage).
+Here's an example from
+the [Framework Agreement Traceability](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_Traceability.pdf)
+(link opens pdf):
+
+| Predefined Policy                  | Typically used where? | Predefined Purpose                                                                                                                                        |
+|------------------------------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `purpose.trace.v1.qualityanalysis` | Notification API      | The data can be used for quality analysis to identify and select affected components and to send quality notifications to affected customers or suppliers |
+
+A `odrl:Constraint` referencing this purpose looks like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "odrl:leftOperand": "https://w3id.org/catenax/policy/UsagePurpose",
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "purpose.trace.v1.qualityanalysis"
+}
+```
```

Comment:
should we make the examples easier to read with adding the remote context from odrl?

```
{
  "@context": {
    "https://www.w3.org/ns/odrl.jsonld"
  },
  "leftOperand": "https://w3id.org/catenax/policy/UsagePurpose",
  "operator": "eq",
  "rightOperand": "purpose.trace.v1.qualityanalysis"
}
```

Reply:
purpose.trace.v1.qualityanalysis -> outdated